staffing/views.py

```diff
@@ -594,6 +607,7 @@ def prod_report(request, year=None, month=None):
     return render(request, "staffing/prod_report.html",
                   {"data": data,
                    "months": months,
+                   "today": date.today(),
```

Comment:
we could declare today one time at the begining, as we use it a lot in this method.